### PR TITLE
chore: Checking `github-action` variable to see auto-merge functional…

### PR DESCRIPTION
**Pull request by awais786**
Checking `github-action` variable to see auto-merge functionality is enabled.

Go to following repo URL and add variable with its value.

**https://github.com/openedx/edx-platform/settings/variables/actions/AUTOMERGE_PYTHON_DEPENDENCIES_UPGRADES_PR**

Add a variable and its value to True

`AUTOMERGE_PYTHON_DEPENDENCIES_UPGRADES_PR` and value to `True`

### CODE REVIEW

**Comment by awais786** [comment]
Screenshot 2023-03-19 at 12 14 23 AM

File: jenkins/github_helpers.py [hunk 1]
```diff
@@ -284,16 +286,35 @@ def verify_upgrade_packages(self, pull_request):
             self._add_comment_about_reqs(pull_request, "List of packages in the PR without any issue", valid_reqs)
 
             if not suspicious_reqs and valid_reqs:
-                # right now this scripts is adding labels on all prs and other scripts
-                # merge these labeled prs. We need to do this only for under arch-bom ownership repos.
-                # pull_request.set_labels('Ready to Merge')
-                logger.info("Total valid upgrades are %s", valid_reqs)
+                if self.check_automerge_variable_value(location):
```
Comment (UsamaSadiq) [inline jenkins/github_helpers.py:289]:
```suggestion
                if self.is_automerge_enabled(location):
```

File: jenkins/github_helpers.py [hunk 2]
```diff
@@ -284,16 +286,35 @@ def verify_upgrade_packages(self, pull_request):
...
+                    pull_request.set_labels('Ready to Merge')
+                    logger.info("Total valid upgrades are %s", valid_reqs)
             else:
                 self._add_comment_about_reqs(pull_request, "These Packages need manual review.", suspicious_reqs)
 
         else:
             logger.info("No package available for comparison.")
 
+    def check_automerge_variable_value(self, location):
```
Comment (UsamaSadiq) [inline jenkins/github_helpers.py:298]:
```suggestion
    def is_automerge_enabled(self, location):
```

**Review by UsamaSadiq — COMMENTED** [review]
Some minor suggestions but otherwise looks good.

**Review by UsamaSadiq — APPROVED** [review]

**Pull request merged**